docs/DeveloperGuide.adoc

```diff
@@ -1219,29 +1276,30 @@ Use case resumes at step 4.
 2. CRM Book shows the requested list
 3. User finds and selects the Lead to convert
 4. CRM Book determines if Lead can be converted and converts if possible
-
 +
 Use case ends.
 
 *Extensions*
 
 [none]
-* 2a. List is empty.
-+
-Use case ends.
-
-* 3a. User selected a Contact, not a Lead.
+* 1a. User entered the Lead information incorrectly.
+* 1b. User entered the add command incorrectly.
 +
 [none]
-** 3a1. CRM Book displays an error message.
+** 1a1. CRM Book displays error message.
 +
-Use case resumes at step 2.
+Use case ends.
 
-* 3a. The given index is invalid.
+* 2a. Lead entered has a phone number or email address stored earlier.
+** 2a1. CRM Book displays message that a possible duplicate detected.
 +
-** 3a1. CRM Book displays an error message.
+Use case ends.
+
+* 2b. Lead entered does not have a phone number or email address stored earlier.
+** 2b1. CRM Book adds Lead.
+** 2b2. CRM Book displays message of successful addition of Lead.
```

Comment:
Similarly, MSS

Reply:
I've updated my part. I think this section belongs to Woody?

Reply:
@zhuleyan But it's in your PR. Can you look through your PR and remove the things that you did not add

Reply:
fixed now